fapp/src/pages/CreateExpense/CreateExpense.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './CreateExpense.css';
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { v4 as uuid } from 'uuid';
+
+const url = 'http://localhost:3030/expenses/'
+const user_id = 1;
+// const locales = 'pl-PL'
+
+const CreateExpense = () => {
+    // const currentDate = new Date();
+    // const formattedDate = currentDate.toLocaleString(locales);
+    // console.warn(formattedDate)
+
+    const [value, setValue] = useState('');
+    const [date, setDate] = useState('');
+    const navigate = useNavigate();
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const new_uuid = uuid();
+        const expense =
+            {
+                id: new_uuid,
+                user_id: user_id,
+                value: value,
+                date: date,
+                tags: [] // todo tags
+            };
+
+        fetch(url, {
+            method: 'POST',
+            headers: {"Content-Type": "application/json"},
+            body: JSON.stringify(expense)
+        }).then(() => {
+            navigate('/expenses/' + new_uuid);
+        })
```

Comment:
Same as before, it would be good to extract the creation logic to a different file i think, to make it easier on us the next sprint